packages/tinybird/src/publish.ts

```diff
@@ -30,6 +30,7 @@ const tinybirdEmail = z.object({
   sizeEstimate: z.number().nullish(), // Estimated size in bytes
 });
 export type TinybirdEmail = z.infer<typeof tinybirdEmail>;
+export type IndexedDBEmail = z.infer<typeof tinybirdEmail>;
```

Comment:
This is the tinybird package and has nothing to do with indexeddb. Please undo this change.

Reply:
Addressed.